docs/Actions.md

```diff
@@ -1321,6 +1321,24 @@ end
 
 ## Misc
 
+### appledoc
+
+Generate Apple-like source code documentation from specially formatted source code comments.
+
+```ruby
+appledoc(
+      project_name: "MyProjectName",
+      project_company: "Company Name",
+      company_id: "com.company.id",
+      input: "MyProjectSources",
+      output: "/tmp/appledoc/myProjectDocs",
```

Comment:
Can we remove the optional parameters, like `output`, `options`, `warnings` and `exit_threshold` here? Would it still work, right?